docs/pages/reference/access-controls/roles.mdx

```diff
@@ -142,6 +144,55 @@ spec:
       'workload': ['database', 'backup']
 ```
 
+### client_idle_timeout
```

Comment:
```suggestion
### Client idle timeout
```
To make the formatting consistent with other headings (regular prose instead of a field name)